translatorbot/daily-llm.py

```diff
@@ -137,7 +137,7 @@ def configure_daily(self):
 
     def call_joined(self, join_data, client_error):
         self.print_debug(f"call_joined: {join_data}, {client_error}")
-        self.client.start_transcription()
+        self.client.start_transcription(settings={"model": "nova-2-ea" })
```

Comment:
We may want to have a fallback to "nova" here, because users may not all have access to Nova 2.